app/helpers/self_service/breadcrumbs_helper.rb

```diff
@@ -62,5 +66,9 @@ def crumb_firm_adviser_new
     def crumb_trading_name_new
       { name: t('self_service.trading_name_new.breadcrumb', firm_name: @firm.registered_name) }
     end
+
+    def crumb_principal_edit
+      { name: t('self_service.principal_edit.breadcrumb', firm_name: @firm.registered_name) }
+    end
   end
 end
```

Comment:
Mmm, good question. I might have missed a spec I needed to add to. @munckymagik?